tools/pubsys-config/src/lib.rs

```diff
@@ -111,6 +111,10 @@ impl InfraConfig {
 #[derive(Debug, Default, Deserialize, Serialize, PartialEq, Eq, Clone)]
 pub struct Vendor {
     pub registry: String,
+    // needed in cases where the vendor does not use HTTPS. this is to accomodate a weird quirk in
```

Comment:
Not a blocker, but i'm not sure we really want to have this be an option for production release, I could see AppSec maybe throwing up a red flag on this?